ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -646,6 +646,9 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
     Message request = ParseMessageMemory(messageMemory, messageSize);
     const MessageHeader* requestHeader = request.messageHeader;
     
+    const char* absolutePath = nullptr;
+    const char* relativePath = nullptr;
```

Comment:
I very much like the naming change here to keep it as absolutePath/relativePath all the way thru.  Very clear!